software/base/src/main/java/org/apache/brooklyn/entity/software/base/VanillaWindowsProcessWinRmDriver.java

```diff
@@ -76,10 +77,13 @@ public void customize() {
 
     @Override
     public void launch() {
-        executeCommandInTask(
```

Comment:
Counter to the launch command, what do you think about making install command obligatory?